src/main/kotlin/de/itemis/mps/gradle/RunAntScript.kt

```diff
@@ -41,6 +43,10 @@ open class RunAntScript : DefaultTask() {
             }
         }
 
+        if(logging.level != LogLevel.LIFECYCLE && !allArgs.any { it.contains("mps\\.ant\\.log") }) {
```

Comment:
Sorry, missed this the first time I reviewed the PR: the argument to `contains` seems to be a regex here, but `contains()` takes a simple string. Perhaps even `.startsWith("-Dmps.ant.log=")` would have been a more precise condition?